pkgs/applications/graphics/displaycal/default.nix

```diff
@@ -0,0 +1,74 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, xorg
+, certifi
+, dbus-python
+, build
+, wxPython_4_1
+, send2trash
+, PyChromecast
+, distro
+, makeWrapper
+, argyllcms
+, gtk3
+, gsettings-desktop-schemas
+}:
+
+buildPythonApplication rec {
+  pname = "displaycal";
+  version = "3.9.10";
+
+  src = fetchPypi {
+    pname = "DisplayCAL";
+    inherit version;
+    sha256 = "sha256-oDHDVb0zuAC49yPfmNe7xuFKaA1BRZGr75XwsLqugHs=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  propagatedBuildInputs = [
+    certifi
+    dbus-python
+    build
+    wxPython_4_1
+    send2trash
+    PyChromecast
+    distro
+  ];
+
+  buildInputs = with xorg; [
+    libX11
+    libXxf86vm
+    libXext
+    libXinerama
+    libXrandr
+  ];
+
+  pythonImportsCheck = [ "DisplayCAL" ];
+
+  doCheck = false; # Tests try to access an X11 session.
+
+  # no idea why it looks there - symlink .json lang (everything)
+  postInstall = ''
+    for x in $out/share/DisplayCAL/*; do
+      ln -s $x $out/lib/python3.10/site-packages/DisplayCAL
+    done
+
+    for binary in "$out/bin/"*; do
```

Comment:
You can set `makeWrapperArgs` for a Python package. It takes a list of arguments you would give to `wrapProgram` (such as `--prefix ..`)

Reply:
Is that an attribute in the set for `buildPythonApplication`

Reply:
It's an environment variable that the `wrapPython` hook listens to.